atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/OracleConnectionConfig.java

```diff
@@ -194,22 +195,22 @@ protected final void check() {
         if (getProtocol() == ConnectionProtocol.TCPS) {
             Preconditions.checkArgument(
                     getTruststorePath().isPresent(), "ConnectionProtocol.TCPS requires a truststore");
-            com.google.common.base.Preconditions.checkArgument(
+            Preconditions.checkArgument(
                     new File(getTruststorePath().get()).exists(),
-                    "truststore file not found at %s",
-                    getTruststorePath().get());
+                    "truststore file not found",
+                    UnsafeArg.of("truststorePath", getTruststorePath().get()));
```

Comment:
On one hand this should normally come from config and so should normally be safe. There isn't necessarily a guarantee in the code that it does, though, and so I think this is probably a safer decision.